Sources/MapboxNavigation/FollowingCameraOptions.swift

```diff
@@ -5,21 +5,23 @@ import CoreLocation
  `NavigationViewportDataSource` in `NavigationCameraState.following` state.
  */
 public struct FollowingCameraOptions: Equatable {
-    
+
     /**
      Pitch, which will be taken into account when preparing `CameraOptions` during active guidance
      navigation.
      
      Defaults to `45.0` degrees.
+
+     - Invariant: Acceptable range of values is `0...85`.
      */
     public var defaultPitch: Double = 45.0 {
         didSet {
             if defaultPitch < 0.0 {
-                preconditionFailure("Lower bound of the pitch should not be lower than 0.0")
+                defaultPitch = 0
```

Comment:
Can we use `assertionFailure` here?

Reply:
It will break tests as well for me.

Reply:
Can we disable asserts in tests? Like `-assert-config Release` in "Other Swift Flags".

Reply:
I'd really like to keep assertions or preconditions. Right now we're basically silently modifying value in case if it's not valid, this breaks the contract.

Reply:
> Can we disable asserts in tests? Like -assert-config Release in "Other Swift Flags".

Can you explain how to use this with SPM?

Reply:
> Can you explain how to use this with SPM?

Not really( 

Maybe we should use wrapped asserts that can be disabled in tests in runtime? I agree with @MaximAlien that it should be possible to use asserts/preconditions in the SDK.

Reply:
> this breaks the contract.

the contact says that acceptable values are `0...85`, so, there is no a violation. 

> that it should be possible to use asserts/preconditions in the SDK.

ok, I will remove tests which cause a crash because it blocks my development and we can fix this separately.

Reply:
#3354